### ISSUE TITLE
Background map error

### ISSUE THREAD

**Issue by MYflipper** [zdila/gpx-animator#78]
When I select Open StreetMap (German Style) as map and click on Render the error message "Error while rendering" appears.

If I use "http://tile.openstreetmap.org/{zoom}/{x}/{y}.png" as map I get an error message.
"Error while rendering: error reading tile http://tile.openstreetmap.org/13/4222/2675.png".

When I open "http://tile.openstreetmap.org/13/4222/2675.png" in the browser, the PNG image is displayed.

If I do not select a map, the video will be created without a background map.

**Comment by McPringle** [comment]
This is a duplicate of #76 which was resolved with commit 8495a8cfc531cc6489c0cdd4a9f566748d76a298. The next version of GPX Animation, which will be released end of July, will contain the fix.

As a hotfix, you can just replace `http` with `https` (the error is a HTTP to HTTPS redirect, which your browser follows automatically).

**Comment by MYflipper** [comment]
if i use "https://tile.openstreetmap.org/{zoom}/{x}/{y}.png" instead of "http://tile.openstreetmap.org/{zoom}/{x}/{y}.png" i still get an error message that the PNG image was not found

**Comment by McPringle** [comment]
Hi @MYflipper, I reopened this issue. To reproduce your problem, can you please provide an example GPX file and the GPX Animator settings (use `File` -> `Save as`)? If you don't want to attach it here, you can send it to my private mail address: marcus@fihlon.swiss

**Comment by McPringle** [comment]
Thank you very much @MYflipper for providing the necessary information by email. I was able to reproduce the error. OpenStreetMap was answering the request from GPX Animator with the status code 429 (too many requests). This particular OSM server was under attack and the admin decided to reject all requests without an agent name in the request header. I extended GPX Animator to set a special agent name in the HTTP request and everything worked. The fix will be available in GPX Animator v1.4.
